service/policy/db/queries/dynamic_value_mappings.sql

```
@@ -0,0 +1,220 @@
+                WHERE dynamic_value_mapping_id = sqlc.arg('id') AND action_id = a
+            )
+    ),
+    update_count AS (
+        SELECT COUNT(*) AS cnt
+        FROM mapping_update
+    )
+SELECT cnt
+FROM update_count;
+
+-- name: deleteDynamicValueMapping :execrows
+DELETE FROM dynamic_value_mappings WHERE id = $1;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)